deps/ReactantExtra/BUILD

```
@@ -448,7 +449,13 @@ cc_library(
 "-Wl,-exported_symbol,_ProfilerActivityEnd",
 "-Wl,-exported_symbol,_ReactantFuncSetArgAttr",
 "-Wl,-exported_symbol,_ReactantCudaDriverGetVersion",
-"-Wl,-exported_symbol,_ReactantLLVMParseCommandLineOptions"
+"-Wl,-exported_symbol,_ReactantLLVMParseCommandLineOptions",
+"-Wl,-exported_symbol,_ReactantLLVMParseCommandLineOptions",
+"-Wl,-exported_symbol,_PjRtDeviceGetLocalDeviceId",
+"-Wl,-exported_symbol,_PjRtDeviceGetGlobalDeviceId",
+"-Wl,-exported_symbol,_PjRtDeviceGetLocalHardwareId",
+"-Wl,-exported_symbol,_XLAExecuteSharded",
+"-Wl,-exported_symbol,_ClientGetPlatformName",
```

Missing `_RegisterEnzymeXLACPUHandler` from https://github.com/EnzymeAD/Reactant.jl/pull/647/files#diff-cb71ef6c799b0e097d3bf358a8fe4eaa18729014d1e645e034911f3d7a3b371fR441
```suggestion
"-Wl,-exported_symbol,_ClientGetPlatformName",
"-Wl,-exported_symbol,_RegisterEnzymeXLACPUHandler",
```